fe/fe-core/src/main/java/org/apache/doris/common/NereidsSqlCacheManager.java

```diff
@@ -253,16 +257,28 @@ private boolean tablesOrDataChanged(Env env, SqlCacheContext sqlCacheContext) {
                 return true;
             }
             OlapTable olapTable = (OlapTable) tableIf;
-            long currentTableTime = olapTable.getVisibleVersionTime();
-            long cacheTableTime = scanTable.latestTimestamp;
+            //long currentTableTime = olapTable.getVisibleVersionTime();
+            //long cacheTableTime = scanTable.latestTimestamp;
             long currentTableVersion = olapTable.getVisibleVersion();
             long cacheTableVersion = scanTable.latestVersion;
             // some partitions have been dropped, or delete or updated or replaced, or insert rows into new partition?
-            if (currentTableTime > cacheTableTime
-                    || (currentTableTime == cacheTableTime && currentTableVersion > cacheTableVersion)) {
+            if (currentTableVersion != cacheTableVersion) {
                 return true;
             }
 
+            if (Config.isCloudMode()) {
+                Collection<Long> partitionIds = scanTable.getScanPartitions();
+                List<CloudPartition> partitions = partitionIds.stream()
+                        .sorted()
+                        .map(olapTable::getPartition)
+                        .map(partition -> (CloudPartition) partition)
+                        .collect(Collectors.toList());
+                try {
+                    CloudPartition.getSnapshotVisibleVersion(partitions);
+                } catch (RpcException e) {
+                    throw new RuntimeException(e);
+                }
```

Comment:
add comment to explain what this code block use for

Reply:
sure, will do